RadixWallet/Features/SettingsFeature/ManualAccountRecoveryScan/ManualAccountRecovery+View.swift

```diff
@@ -0,0 +1,176 @@
+				olympiaSection()
+			}
+		}
+		.background(.app.gray5)
+		.toolbar {
+			ToolbarItem(placement: .automatic) {
+				CloseButton {
+					store.send(.view(.closeButtonTapped))
+				}
+			}
+		}
+	}
+
+	private func babylonHeader() -> some View {
+		VStack(spacing: .zero) {
+			Text("Account Recovery Scan") // FIXME: Strings
+				.textStyle(.sheetTitle)
+				.multilineTextAlignment(.center)
+				.foregroundColor(.app.gray1)
+				.padding(.top, .small1)
+				.padding(.horizontal, .medium1)
+				.padding(.bottom, .medium3)
+
+			Text(LocalizedStringKey(text))
+				.textStyle(.body1Regular)
+				.foregroundStyle(.app.gray2)
+				.padding(.bottom, .medium1)
+
+			Text("Babylon Accounts") // FIXME: Strings
+				.textStyle(.sectionHeader)
+				.foregroundStyle(.app.gray1)
+				.padding(.bottom, .medium3)
+
+			Text(LocalizedStringKey("Scan for Accounts originally created on the **Babylon** network:")) // FIXME: Strings
+				.textStyle(.body1Regular)
+				.foregroundStyle(.app.gray2)
+				.flushedLeft
+				.padding(.bottom, .medium3)
+		}
+		.padding(.horizontal, .medium2)
+	}
+
+	private func babylonSection() -> some View {
+		VStack(spacing: 0) {
+			Button("Use Seed Phrase") { // FIXME: Strings - repeated
+				store.send(.view(.useSeedPhraseTapped(isOlympia: false)))
+			}
+			.padding(.bottom, .medium2)
+
+			Button("Use Ledger Hardware Wallet") { // FIXME: Strings - repeated
+				store.send(.view(.useLedgerTapped(isOlympia: false)))
+			}
+		}
+		.buttonStyle(.secondaryRectangular(shouldExpand: true))
+		.padding(.top, .medium2)
+		.padding(.horizontal, .medium1)
+		.padding(.bottom, .large3)
+		.background(.app.white)
+	}
+
+	private func olympiaHeader() -> some View {
+		VStack(spacing: .zero) {
+			Text("Olympia Accounts") // FIXME: Strings
+				.textStyle(.sectionHeader)
+				.foregroundStyle(.app.gray1)
+				.padding(.top, .medium1)
+
+			Text(LocalizedStringKey("Scan for Accounts originally created on the **Olympia** network:")) // FIXME: Strings
+				.textStyle(.body1Regular)
+				.foregroundStyle(.app.gray2)
+				.flushedLeft
+				.padding(.vertical, .medium3)
+		}
+		.padding(.horizontal, .medium2)
+	}
+
+	private func olympiaSection() -> some View {
+		VStack(spacing: 0) {
+			Button("Use Seed Phrase") { // FIXME: Strings - repeated
+				store.send(.view(.useSeedPhraseTapped(isOlympia: true)))
+			}
+			.padding(.bottom, .medium2)
+
+			Button("Use Ledger Hardware Wallet") { // FIXME: Strings - repeated
+				store.send(.view(.useLedgerTapped(isOlympia: true)))
+			}
+			.padding(.bottom, .small2)
+
+			Text(LocalizedStringKey("Note: You will still use the new **Radix Babylon** app on your Ledger device.")) // FIXME: Strings
+				.textStyle(.body1Regular)
+				.foregroundStyle(.app.gray2)
+				.flushedLeft
+		}
+		.buttonStyle(.secondaryRectangular(shouldExpand: true))
+		.padding(.top, .medium2)
+		.padding(.horizontal, .medium1)
+		.padding(.bottom, .small1)
+		.background(.app.white)
+	}
+}
+
+private let text: String =
```

Comment:
So we dont break it later, return `LocalizedStringKey` here instead, so that the bolding works?

Reply:
My plan was to replace `text` at the call site with `L10n.AccountRecovery.xxxx`

Reply:
wrapped in a `LocalizedStringKey(L10n..)` you mean?